backend/codesirius/problems/models/language.py

```
@@ -0,0 +1,20 @@
+
+    A language is a programming language that can be used to solve a problem.
+
+    Attributes:
+        name (str): The name of the language.
+        version (str): The version of the language.
+    """
+
+    name = models.CharField(max_length=50, unique=True)
+    version = models.CharField(max_length=50, blank=True)  # Optional
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)